medusa/search/proper.py

```diff
@@ -123,61 +113,57 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
                                                 b"WHERE date >= ? AND action LIKE '%10'", [last_subtitled])
 
         if not recently_aired:
-            logger.log('No recently aired new episodes, nothing to search for')
+            log.info('No recently aired new episodes, nothing to search for')
             return []
 
         # Loop through the providers, and search for releases
         for cur_provider in providers:
             threading.currentThread().name = '{thread} :: [{provider}]'.format(thread=original_thread_name,
                                                                                provider=cur_provider.name)
 
-            logger.log('Searching for any new PROPER releases from {provider}'.format
-                       (provider=cur_provider.name))
+            log.info('Searching for any new PROPER releases from {provider}', {'provider': cur_provider.name})
 
             try:
                 cur_propers = cur_provider.find_propers(recently_aired)
             except AuthException as e:
-                logger.log('Authentication error: {error}'.format
-                           (error=ex(e)), logger.DEBUG)
+                log.debug('Authentication error: {error}', {'error': ex(e)})
                 continue
             except socket_timeout as e:
-                logger.log('Socket time out while searching for propers in {provider}, skipping: {error}'.format
-                           (provider=cur_provider.name, error=ex(e)), logger.DEBUG)
+                log.debug('Socket time out while searching for propers in {provider}, skipping: {error}',
```

Comment:
Can you remove the ex import and these function calls. And remove them with {error!r}